src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -99,76 +99,85 @@ private static void WriteClientFields(CodeWriter writer, LowLevelClient client)
             writer.Line();
         }
 
+        private static bool IsEmptyConstructorExisted(LowLevelClient client)
+        {
+            foreach (var constructor in client.SecondaryConstructors.Concat(client.PrimaryConstructors))
```

Comment:
You don't need to check `PrimaryConstructors` - all primary constructors have `ClientOptions` parameter